core/controllers/topics_and_skills_dashboard_test.py

```diff
@@ -266,14 +266,154 @@ def test_post_with_status(self):
 
         json_response = self.post_json(
             feconf.SKILL_DASHBOARD_DATA_URL, {
-                'num_items_to_fetch': '10',
+                'num_skills_to_fetch': 10,
                 'status': 'Unassigned'
             }, csrf_token=csrf_token)
 
         self.assertEqual(len(json_response['skill_summary_dicts']), 0)
         self.assertFalse(json_response['more'])
         self.assertEqual(json_response['next_cursor'], None)
 
+    def test_fetch_filtered_skills_with_given_cursor(self):
```

Comment:
Shouldn't you also check things like "two successive fetches don't return the same objects"?

Reply:
Done